guess_the_number.py

```diff
@@ -16,7 +16,9 @@ def generate_secret(low, high):
 
 
 def get_guess():
+    global guess_count
```

Comment:
Do we want a global variable here or should we pass it when calling the functions?